gertrude_base/gertrude.c

```diff
@@ -65,10 +65,12 @@ int main(int ac, char **av)
     }
     ger.epitech = 0;
     ger.variables = NULL;
-    ger.rule_names = NULL;
+    ger.rule_names = malloc(sizeof(NULL));
+    ger.rule_names[0] = NULL;
     ger.deps = NULL;
     ger.cmds = NULL;
-    ger.rules = NULL;
+    ger.rules = malloc(1);
+    strcpy(ger.rules, "\0");
```

Comment:
why not strdup here?

Reply:
I wanted to do that but I forgor, i already did it at some other point in the new rules code. I suggest you add that in the pr.

Reply:
👍